CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.18.0] - 2022-12-dd
```

Comment:
Please add an `## [Unreleased]` header to the top and provide a date when you'd expect this release, it can be 12th or 13th for example.